bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -1889,7 +1911,7 @@ public function doExportAction(Request $request, LocaleServiceInterface $localeS
         $firstLine = true;
         $lineCount = count($csv);
 
-        if (!$addTitles) {
+        if (!$addTitles && $lineCount > 0) {
```

Comment:
If it is a fix, then create a new PR based on 6.8 branch and remove it from here.  thanks!

Reply:
I think this fix is only relevant with the change I made. I can hardly think of a situation in the current version where this would be an issue.